fix(installer): add request and limit resources for cilium workload

yunjianzhong opened the pull request:
> /kind bug

currently the request and limit resources weren't set in the manifests of the cilium workloads. this pull request is target to solve the issue.

Fixes #1312

leoryu commented:
@yunjianzhong your commit message doesn't pass the lint check. In your case, commit message should be `fix(installer): add request and limit resources for cilium workload`.

leoryu commented:
ref https://github.com/tkestack/tke/blob/master/CONTRIBUTING.md

yunjianzhong commented:
> @yunjianzhong your commit message doesn't pass the lint check. In your case, commit message should be `fix(installer): add request and limit resources for cilium workload`.

I just changed the commit message, please check. thanks!

leoryu commented:
> > @yunjianzhong your commit message doesn't pass the lint check. In your case, commit message should be `fix(installer): add request and limit resources for cilium workload`.
> 
> I just changed the commit message, please check. thanks!

I think you missed a space after `:`.

yunjianzhong commented:
> > > @yunjianzhong your commit message doesn't pass the lint check. In your case, commit message should be `fix(installer): add request and limit resources for cilium workload`.
> > 
> > 
> > I just changed the commit message, please check. thanks!
> 
> I think you missed a space after `:`.

got it, changed again.
